### ISSUE TITLE
Text alignments

### ISSUE THREAD

**Issue by zenseii** [teamrocketslime/RS3DS-Releases#17]
### Description
Various texts need to be realigned within their boxes.

- [x] Nr.1 Results in top screen when finishing any level should be more central.
- [x] Nr.2 The enter button "End" when inputting name when starting new game or when connecting with wifi, is misaligned to the left.
- [x] Nr.3 Whenever you've gotten a new ship part the "New" for All ship parts is too far to the right compared to the others categories (Hull, Bow etc.). (See screenshot)
- [ ] Nr.4 Lakeside in bottom screen should be more central in the namebox.
- [x] Nr.5 Hidden Ice 2 and 3 have the numbers at different distances.
- [ ] Nr.6 Zipunk in bottom screen namebox.
- [x] Nr.7 Taoyuki Ueno is misaligned to the left.
- [x] Nr.8 Masaya Tsunemoto is misaligned to the left.
- [ ] Nr.9 Slushia Mines is aligned to the left and not centrally.

### Screenshot
Nr.1
![2019-08-15_14-59-31 564_top](https://user-images.githubusercontent.com/12501091/64501472-cd83d180-d2c1-11e9-9abf-9aaebfdef7a0.png)
Nr.2
![2019-08-25_04-14-30 200_bot](https://user-images.githubusercontent.com/12501091/64501480-d2e11c00-d2c1-11e9-8a51-ba1d3a4f8913.png)
Nr.3
![2019-08-27_06-21-39 845_bot](https://user-images.githubusercontent.com/12501091/64501496-e2606500-d2c1-11e9-8b13-a90d3dd0bf44.png)
Nr.4
![2019-08-31_00-09-31 490_bot](https://user-images.githubusercontent.com/12501091/64501508-f0ae8100-d2c1-11e9-8eba-560bbe92655f.png)
Nr.5
![2019-09-05_05-58-38 372_bot](https://user-images.githubusercontent.com/12501091/64501515-f73cf880-d2c1-11e9-93f1-3522fb2c32d5.png)
Nr.6
![2019-09-06_18-34-45 482_bot](https://user-images.githubusercontent.com/12501091/64501535-0cb22280-d2c2-11e9-8d0a-7c477ca11b2d.png)
Nr.7
![2019-09-08_17-50-55 482_bot](https://user-images.githubusercontent.com/12501091/64501543-150a5d80-d2c2-11e9-9558-1d8a45d9dcd3.png)
Nr.8
![2019-09-08_17-51-53 357_bot](https://user-images.githubusercontent.com/12501091/64501546-18054e00-d2c2-11e9-8d17-bd20ba08c622.png)
Nr.9
![2019-08-27_04-54-30 858_bot](https://user-images.githubusercontent.com/12501091/64501484-d674a300-d2c1-11e9-9253-3bc33b014482.png)

### Additional Info
I didn't know what issue to put this as.

**Comment by ultramario1998** [comment]
- Nos. 2, 3, 7 and 8 have been fixed and will be included in the next build.
- I'm unsure what we're going to do with area name justification. What you see in the current beta is how the Japanese version treats its name textboxes (with size and justification ugliness present). Optimally, we would trim down the name textboxes so that the area names fit nicely in them, but that's much harder than it sounds - not only do we lack any sort of out-of-game visualizer (so we'd need to debug everything in-game), it'd certainly wreak havoc on the tenuous consistency we currently have with the textboxes. (eg we'd have one area's name textbox start at a completely different X-value from another's, and without any real way to compare the two without loading screenshots into Photoshop, we'd have a bad time due to the sheer quantity of boxes that need fixing) Therefore, I'm not going to touch these yet, but it is something we'll keep in mind for future builds.
- With that said, I have applied a quick and dirty fix for No. 1, due to its prevelance.
- I've also passed No. 5 along to our graphics people.
